src/lib/action.rs

```diff
@@ -213,7 +216,7 @@ impl Glue {
     pub fn new(from: NodeId<FromNodeId>, parent: NodeId<FromNodeId>, position: u16) -> Glue {
         Glue { from_node: from,
                node: parent,
-               pos : position}
+               pos: position, }
```

Comment:
The `, ` seems unnecessary?

Reply:
Added by rustfmt!

Reply:
Oh god, can we turn that feature off? I mean, that's just ugly...